frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/GoService.java

```diff
@@ -0,0 +1,90 @@
+package ch.shiftcrypto.bitboxapp;
+
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.Service;
+import android.content.Intent;
+import android.os.Binder;
+import android.os.Build;
+import android.os.IBinder;
+
+import androidx.core.app.NotificationCompat;
+
+import goserver.GoAPIInterface;
+import goserver.GoEnvironmentInterface;
+import goserver.Goserver;
+
+public class GoService extends Service {
+    // Binder given to clients
+    private final IBinder binder = new GoServiceBinder();
+
+    private Boolean started = false;
```

Comment:
why not lowercase `boolean`?

Reply:
No reason, my Java is just  bit rusted (no pun intended :stuck_out_tongue: )